docs/conf/prague/2019/news/cfp-diversity-tickets.rst

```diff
@@ -0,0 +1,47 @@
+:template: {{year}}/generic.html
+
+.. post:: Apr 25, 2019
+   :tags: prague-2019, tickets, cfp, diversity
+
+Call for Proposals Closing Soon, Diversity Tickets Available
+============================================================
+
+Greetings, fellow documentarians in Europe and beyond! In this post we're happy to announce our diversity scholarship initiative and remind you that the call for proposals is about to close, so if you want to submit your proposal now is the time.
```

Comment:
Perhaps only saying "diversity ticket" instead of "diversity scholarship" is better? The word scholarship makes me feel like more costs are covered.